src/services/surveyInterfaces.ts

```diff
@@ -42,6 +42,7 @@ enum RatingType {
   STAR = 'star',
   HEART = 'heart',
   SMILEY = 'smiley',
+  MONEY = 'money',
```

Comment:
Should we order this list alphabetically? 🤔 
(super minor suggestion as the list remains quite short :) )